sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTablesSuite.scala

```diff
@@ -138,4 +121,21 @@ class ShowTablesSuite extends ShowTablesSuiteBase with CommandSuiteBase {
       }
     }
   }
+
+  test("SPARK-34157 Unify output of SHOW TABLES and pass output attributes properly") {
```

Comment:
can we put this test into the base test suite? so that it can be run with different catalog implementations.

Reply:
OK